VS_Project/Pokerapp/Pokerapp/delete-ajax.cshtml

```diff
@@ -0,0 +1,48 @@
+﻿@using WebMatrix.Data
```

Comment:
Waarom maak je hier een cshtml pagina voor aan? Kan dit niet gewoon in een class / javascript file / de pagina waar je in werkt? Want mensen kunnen nu deze pagina aanroepen door de url in te typen.